cmd/lncli/commands.go

```diff
@@ -719,6 +720,11 @@ var closeChannelCommand = cli.Command{
 			Usage: "the output index for the funding output of the funding " +
 				"transaction",
 		},
+		cli.StringFlag{
+			Name: "chan_point",
+			Usage: "(optional) the channel point. If set, funding_txid " +
```

Comment:
nit: wrap code at 80 characters (while using a tabulator width of 8 characters in your editor), here and a few other places in your commits.

Reply:
corrected